### ISSUE TITLE
File download example does not work

### ISSUE THREAD

**Issue by benjaminsaljooghi** [ncbi/datasets#25]
The file download example from the [documentation](https://www.ncbi.nlm.nih.gov/datasets/docs/command-line-assembly/):

```
# The text file (3_mammals.txt) contains one accession per row:
$ cat 3_mammals.txt
GCA_001922835.1
GCF_009762305.2
GCA_004348235.1

# Use the --inputfile flag to download the genome dataset using the text file to specify the desired genomes
$ ./datasets download genome accession --inputfile 3_mammals.txt --filename marine_genomes.zip 
```

Does not work, it gives me this error on Windows:

```
Error: requires at least 1 arg(s), only received 0
```

**Comment by BradHolmes** [comment]
Thank you for the report!  We are aware of this bug, and intend to get a fix out today.  Sorry for the trouble,

**Comment by BradHolmes** [comment]
The just released CLI version 10.11.0 has fixed this problem
